package.json

```diff
@@ -105,7 +105,10 @@
       }
     },
     "rules": {
-      "arrow-parens": ["error", "always"],
+      "arrow-parens": [
+        "error",
+        "always"
+      ],
```

Comment:
Oh if these are in master I missed them somewhere 😊 you can leave them in in that case